mediabridge/data_processing/wiki_to_netflix.py

```diff
@@ -209,50 +188,67 @@ def wiki_query(data_csv, user_agent):
         response.raise_for_status()
         data = response.json()
 
-        wiki_movie_ids.append(wiki_feature_info(data, "item"))
-        wiki_genres.append(wiki_feature_info(data, "genreLabel"))
-        wiki_directors.append(wiki_feature_info(data, "directorLabel"))
+        wiki_data_list.append(
+            MovieData(
+                movie_id=wiki_feature_info(data, "item"),
+                genre=wiki_feature_info(data, "genreLabel"),
+                director=wiki_feature_info(data, "directorLabel"),
+            )
+        )
 
-    return wiki_movie_ids, wiki_genres, wiki_directors
+    return wiki_data_list
 
 
 def process_data(test=False):
     """
-    Reads movie titles and release years from the Netflix data set (which should be downloaded and placed in the repo),
-    then tries to match them with data from Wikidata. For any matches, a CSV file is written.
+    Processes Netflix movie data by enriching it with information from Wikidata and writes the results to a CSV file.
+    Netflix data was conveted from a generator to a list to avoid exaustion. was running into an issue where nothing would print to CSV file
     """
     missing_count = 0
     processed_data = []
 
-    netflix_data = read_netflix_txt(os.path.join(data_dir, "movie_titles.txt"), test)
+    netflix_data = list(
```

Comment:
This is fine for now, but instead of "rendering" the generator into a list (so that it can be iterated through multiple times, which is what it seems we are doing below), we should instead write the code so that it can operate on the data iteratively and not have to reference the rows multiple times. No need to do that here, something to keep in mind for a future PR.

Reply:
Also nice job figuring out why the CSV wasn't being written (though in the big picture, we don't need the CSV).